apiserver/facades/controller/caasunitprovisioner/state.go

```diff
@@ -67,6 +67,7 @@ type Model interface {
 // required by the CAAS unit provisioner facade.
 type Application interface {
 	GetScale() int
+	Scale(int) error
```

Comment:
`SetScale` is a better name?

Reply:
Imported from existing state method so I changed the method nae across the stack